packages/element/jsx-dev-runtime/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Do we need these separate package.json file? Can't we use the `@wordpress/element` package.json and output multiple scripts instead to support `import _jsx from @wordpress/element/jsx-runtime`?

Reply:
I think [subpath exports](https://nodejs.org/docs/latest-v12.x/api/packages.html#packages_subpath_exports) is marked experimental in Node 12. Using multiple `package.json` is a more compatible way and is used by `preact`, `@emotion/react`, etc.

Reply:
Maybe we even don't need to use `@wordpress/element` to proxy those files anymore?

If you look at source code of those files:
https://unpkg.com/browse/react@17.0.2/cjs/react-jsx-dev-runtime.development.js
https://unpkg.com/browse/react@17.0.2/cjs/react-jsx-runtime.development.js

The only reference to React is:

```js
var ReactSharedInternals = React.__SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED;
```

Given that those files don't use any public API from React it's probably fine to use those files directly.